snuba/cli/bulk_load.py

```diff
@@ -10,15 +11,24 @@
 
 @click.command()
 @click.option(
-    "--dataset", type=click.Choice(DATASET_NAMES), help="The dataset to bulk load"
+    "--dataset",
+    "dataset_name",
+    type=click.Choice(DATASET_NAMES),
+    help="The dataset to bulk load",
 )
 @click.option(
     "--source",
     help="Source of the dump. Depending on the dataset it may have different meaning.",
 )
 @click.option("--dest-table", help="Clickhouse destination table.")
 @click.option("--log-level", default=settings.LOG_LEVEL, help="Logging level to use.")
-def bulk_load(dataset, dest_table, source, log_level):
+def bulk_load(
+    *,
+    dataset_name: Optional[str],
+    dest_table: Optional[str],
+    source: Optional[str],
+    log_level: str,
+) -> None:
```

Comment:
These types reflect the current option behavior, but do cause these new type errors:

```
snuba/cli/bulk_load.py:43: error: Argument 1 to "get_dataset" has incompatible type "Optional[str]"; expected "str"
snuba/cli/bulk_load.py:47: error: Argument "path" to "load" of "PostgresSnapshot" has incompatible type "Optional[str]"; expected "str"
```

Reply:
in the confirm_load script the option is defined in the same way but you provided str not Optional[str]. Is that intentional ?

Reply:
That was an oversight on my part: this file (`bulk_load`) is correct, the other one (`confirm_load`) should be fixed to mirror this approach.

Reply:
Corrected the optionality of the function arguments here: ba4d0e05e0475badfae2f473ed6352481b937460